src/DiffSharp.Core/Extensions.fs

```diff
@@ -43,53 +43,53 @@ module Array =
 /// Contains extensions to the F# Seq module. 
 module Seq =
 
-    /// Get the index of the maximum element of the sequence
+    /// Gets the index of the maximum element of the sequence.
     let maxIndex seq =  seq |> Seq.mapi (fun i x -> i, x) |> Seq.maxBy snd |> fst
 
-    /// Get the index of the minimum element of the sequence
+    /// Gets the index of the minimum element of the sequence.
```

Comment:
.NET docs are usually written "Get", "Return"  etc. but I'm fine with this form.  You may have to keep an eye on me as it's a matter of habit to do the former :-)